sql/step_XX_remove_old_admission_fields.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE OH_ADMISSION 
```

Comment:
change `XX` to `85`

Reply:
yes, I will do just before the merge, thanks!